src/utils/strapiAPIs.ts

```diff
@@ -0,0 +1,8 @@
+// This file mocks dissecting the data that would normally be returned from strapi. I'd approach this a little differently if actually connected to Strapi and running fetches
+import { header } from "./mockStrapiResponses";
+
+
+export function fetchHeader() {
+  const headerResponse = header.data.attributes.title;
```

Comment:
If you wanted to be fancy, you could just pull out the title and return it:

`const { title } = header.data.attributes;`

Not required though